doc/articles/faq.md

```diff
@@ -0,0 +1,268 @@
+
+## XAML Edit and Continue does not work
+
+You need to make sure that:
+- Your Windows project's 
+	- target SDK version is above Fall Creators Update (16299)
+	- has been built once successfully
+	- the `Microsoft.NETCore.UniversalWindowsPlatform` package version is **above 6.0**
+- That you've closed all XAML editor Windows after having build the application
+- That the project selector at the left of the XAML editor is selecting your windows project
+
+If you're still having issues, **restart visual studio** and/or **unload the iOS/Android and WASM projects**, using right-click on the project, then **unload**.
+
+## Is Uno.UI's Performance on WebAssembly going to improve
+Yes! The current performance is bound the runtime interpreted mode of mono-wasm. The Mono is working on implementing **AOT** compilation, which will improve significantly the performance. See [Miguel de Icaza's status update](https://gitter.im/aspnet/Blazor?at=5b1ab670dd54362753f8a168) for more details.
+
+## Does Uno.UI support macOS ?
+
+Not yet. We're still in a wait and see posture, until Apple's Marzipan cross macOS-iOS APIs become available.
+
+## Does Uno.UI support Linux ?
+
+No, not at the moment. Our assumption for now is that the WebAssembly part of Uno.UI will be efficient for it to be a viable alternative, but if you find that native support still is a viable scenario, please open a GitHub issue.
+
+## Getting started with uwp?
+
+Visit Microsoft's documentation for a primer on UWP : https://docs.microsoft.com/en-us/windows/uwp/get-started/
+
+## Error: `System.SystemException: Thread creation failed.` on WebAssembly
+
+If you're getting this error:
+```
+Error: System.Threading.Tasks.TaskSchedulerException: An exception was thrown by a TaskScheduler. ---> System.SystemException: Thread creation failed.
+at System.Threading.Thread.StartInternal (System.Security.Principal.IPrincipal principal, System.Threading.StackCrawlMark& stackMark) <0x1a68870 + 0x00028> in <bd95bcd953c94273b1c4cdb69e9b2632>:0
+	Microsoft.Extensions.Logging.ConsoleLoggerExtensions.AddConsole (Microsoft.Extensions.Logging.ILoggerFactory factory, System.Func`3[T1,T2,TResult] filter, System.Boolean includeScopes)
+```
+
+Keep the version 1.1.1 of Microsoft.Extensions.Logging; latest version of Logging Extensions is starting a new thread and it's not supported in Wasm.
+
+## I created my first solution and ran the uwp project. Then I wanted to test the magic so I changed the "Hello world" text, hit save and... no live update. Am I missing something?
+
+For live update on UWP you need to edit the xaml while the project is debugging, no need for save, it updates on every keystroke (more or less) you need to update to the latest UWP sdk in your project configuration, change target version to latest
+
+## if a 3rd party has some plugins for uwp, would they render on devices?
+
+If they are being recompiled on top of Uno.UI, they could be running yes.
```

Comment:
I don't understand this question or the answer, can this be rephrased more clearly?